pom.xml

```diff
@@ -178,6 +178,23 @@
         </dependency>
     </dependencies>
 
+<!--    <reporting>
```

Comment:
Can this bit be removed?

Reply:
Sorry for adding this, it’s for personal static code analysis purposes. Feel free to remove it